meson.build

```diff
@@ -3,7 +3,7 @@ project(
   'c',
   license: 'LGPL-2.1-or-later',
   meson_version: '>= 0.56.2',
-  version: '1.4.2',
+  version: '1.5.0',
```

Comment:
Would it perhaps make sense to version the development branch as `1.5.0-alpha`, `1.5.2-alpha`, ..., `1.5.n-rc` as proposed in flatpak/flatpak#6014?

Reply:
At least I don't have a preference either way

The only thing I want, is whatever it is a) the scheme should be documented in README b) consistent between flatpak and flatpak-builder.

Reply:
Also, off-topic for this PR but note that distros/people often switch to unstable when some featurework they need lands, IIRC for Flatpak the curl http backend landed in 1.13.x and libsoup2 became deprecated around that time.

Similarly, here libappstream port landed in 1.3.4 and appstream-glib was deprecated for a long time. 

I'm not sure if adding some `-alpha, -beta` suffix is going to prevent people from doing that.

fdsdk was also tracking Flatpak and Flatpak-builder unstable releases for a long time until I switched it to stable last year. I wasn't around when they started using unstable so I don't know the exact reasoning behind that but I do know that fdsdk maintainers are/were aware of the odd-even versioning rule.